ramalama/model.py

```diff
@@ -175,7 +175,7 @@ def setup_container(self, args):
         if hasattr(args, "port"):
             conman_args += ["-p", f"{args.port}:{args.port}"]
 
-        if sys.platform == "darwin" or os.path.exists("/dev/dri"):
+        if sys.platform == "darwin" and os.path.exists("/dev/dri"):
```

Comment:
I don't think /dev/dri exists in macOS. Do you have it? 

```
 sw_vers 
ProductName:		macOS
ProductVersion:		15.2
BuildVersion:		24C101
$ ls /dev/dri
ls: /dev/dri: No such file or directory
```

Reply:
Correct, the path does not exist on Mac with Docker. I was trying to fix the issue by changing from or to and because the or was short circuiting since the platform check is first. I realized that isn't the right fix. I was also not entirely sure why this check for platform was here and combined with the existence check for the path since this path did not exist on Mac I felt like I was missing something.

Reply:
Correct, the path does not exist on Mac with Docker. I was trying to fix the issue by changing from or to and because the or was short circuiting since the platform check is first. I realized that isn't the right fix. I was also not entirely sure why this check for platform was here and combined with the existence check for the path since this path did not exist on Mac I felt like I was missing something.

Reply:
@volker48 all good, just update the patch as soon you test the new logic, thanks for reaching out and contributing.

Reply:
/dev/dri exists if you use the podman-machine hypervisor with GPU acceleration krunkit.

It's a funny problem because you can't really check if /dev/dri exists on macOS (where this python code runs). But we want GPU acceleration with podman at least.

I believe docker has this GPU acceleration feature also, they used krunkit written by @slp and called it something else.

So another possible fix is to detect if docker on macOS is using krunkit. But the main thing is, we don't want to break accelerated GPU support with podman on macOS here.

Reply:
Thanks @ericcurtin I installed Podman and can see the code is working fine there. Let me update my approach so I don't break anything.